### PR TITLE
Various UI fixes

### PR DESCRIPTION
**Pull request by Faalagorn**
* Unified capitalisation
* Dots after perk descriptions
* Other minor fixes

### CODE REVIEW

File: EN/UI_EN.txt [hunk 1]
```diff
@@ -252,9 +252,9 @@ UI_EN = {
 	UI_trait_brave = "Brave",
 	UI_trait_bravedesc = "Less prone to becoming panicked.",
 	UI_trait_SpeedDemon = "Speed Demon",
-	UI_trait_SpeedDemonDesc = "The fast driver",
-	UI_trait_SundayDriver = "Sunday driver",
-	UI_trait_SundayDriverDesc = "The very slow driver",
+	UI_trait_SpeedDemonDesc = "Drives very fast.",
+	UI_trait_SundayDriver = "Sunday Driver",
+	UI_trait_SundayDriverDesc = "Drives very slow.",
```
Comment (Mishasama) [inline EN/UI_EN.txt:257]:
```suggestion
	UI_trait_SpeedDemonDesc = "The fast driver.",
	UI_trait_SundayDriver = "Sunday Driver",
	UI_trait_SundayDriverDesc = "The very slow driver.",
```
It means different things. It is a noun, not an adjective. I don't think it is necessary to change it.

File: EN/UI_EN.txt [hunk 2]
```diff
@@ -570,23 +570,23 @@ UI_EN = {
     UI_Intro3 = "THIS IS HOW YOU DIED",
     UI_mainscreen_beginner = "BEGINNER",
     UI_trait_FastHealer = "Fast Healer",
-    UI_trait_FastHealerDesc = "Recovers quickly from injuries and illness",
+    UI_trait_FastHealerDesc = "Recovers quickly from injuries and illness.
```
Comment (Mishasama) [inline EN/UI_EN.txt:573]:
```suggestion
    UI_trait_FastHealerDesc = "Recovers quickly from injuries and illness.",
```

File: EN/UI_EN.txt (hunk 1 above)
Comment (Faalagorn) [reply EN/UI_EN.txt:257]: Thanks! I know the sentence changes here, but that's exactly why I wanted to change it. :) It's a description for a perk that makes you drive faster or slower (I don't know the exact percent), so I found it mor fitting as an adjective.

File: EN/UI_EN.txt [hunk 3]
```diff
@@ -79,9 +79,9 @@ UI_EN = {
 	UI_optionscreen_player2 = "Player 2"
 	UI_optionscreen_player3 = "Player 3"
 	UI_optionscreen_player4 = "Player 4"
-	UI_optionscreen_context_menu_font = "Context-menu font"
-	UI_optionscreen_inventory_font = "Inventory font"
-	UI_optionscreen_tooltip_font = "Tooltip font"
+	UI_optionscreen_context_menu_font = "Context Menu Font"
+	UI_optionscreen_inventory_font = "Inventory Font"
+	UI_optionscreen_tooltip_font = "Tooltip Font"
```
Comment (Mishasama) [inline EN/UI_EN.txt:84]:
```suggestion
	UI_optionscreen_context_menu_font = "Context Menu Font",
	UI_optionscreen_inventory_font = "Inventory Font",
	UI_optionscreen_tooltip_font = "Tooltip Font",
```

File: EN/UI_EN.txt [hunk 4]
```diff
@@ -79,9 +79,9 @@ UI_EN = {
 	UI_optionscreen_player2 = "Player 2"
 	UI_optionscreen_player3 = "Player 3"
 	UI_optionscreen_player4 = "Player 4"
```
Comment (Mishasama) [inline EN/UI_EN.txt:81]:
```suggestion
	UI_optionscreen_player2 = "Player 2",
	UI_optionscreen_player3 = "Player 3",
	UI_optionscreen_player4 = "Player 4",
```

File: EN/UI_EN.txt [hunk 5]
```diff
@@ -91,19 +91,19 @@ UI_EN = {
 	UI_optionscreen_language = "Language",
 	UI_optionscreen_keybinding = "KEY BINDINGS",
 	UI_optionscreen_audio = "AUDIO"
```
Comment (Mishasama) [inline EN/UI_EN.txt:93]:
```suggestion
	UI_optionscreen_audio = "AUDIO",
```

File: EN/UI_EN.txt [hunk 6]
```diff
@@ -91,19 +91,19 @@ UI_EN = {
 	UI_optionscreen_language = "Language",
 	UI_optionscreen_keybinding = "KEY BINDINGS",
 	UI_optionscreen_audio = "AUDIO"
-	UI_optionscreen_sound_volume = "Sound volume"
-	UI_optionscreen_music_volume = "Music volume"
+	UI_optionscreen_sound_volume = "Sound Volume"
+	UI_optionscreen_music_volume = "Music Volume"
```
Comment (Mishasama) [inline EN/UI_EN.txt:95]:
```suggestion
	UI_optionscreen_sound_volume = "Sound Volume",
	UI_optionscreen_music_volume = "Music Volume",
```

File: EN/UI_EN.txt [hunk 7]
```diff
@@ -91,19 +91,19 @@ UI_EN = {
...
+	UI_optionscreen_flies_volume = "Flies Volume",
 	UI_optionscreen_flies_volume_tt = "The sound of flies buzzing around corpses.",
-	UI_optionscreen_heart_volume = "Heart volume",
+	UI_optionscreen_heart_volume = "Heart Volume",
 	UI_optionscreen_heart_volume_tt = "The two major sounds heard in the normal heart sound like 'lub dub'. <BR> The 'lub' is the first heart sound, commonly termed S1, and is caused by turbulence caused by the closure of mitral and tricuspid valves at the start of systole. <BR> The second sound, 'dub' or S2, is caused by the closure of aortic and pulmonic valves, marking the end of systole.",
-	UI_optionscreen_vehicle_engine_volume = "Vehicle engine volume",
+	UI_optionscreen_vehicle_engine_volume = "Vehicle Engine Volume",
 	UI_optionscreen_vehicle_engine_volume_tt = "Vroom! Vroom! VROOM!",
 	UI_optionscreen_controller = "CONTROLLER",
 	UI_optionscreen_controller_tip = "Check each controller the game should use."
```
Comment (Mishasama) [inline EN/UI_EN.txt:109]:
```suggestion
	UI_optionscreen_controller_tip = "Check each controller the game should use.",
```

File: EN/UI_EN.txt [hunk 8]
```diff
@@ -553,7 +553,7 @@ UI_EN = {
     UI_Scoreboard_Kick = "Kick",
     UI_Scoreboard_Invisible = "Invisible",
     UI_Scoreboard_GodMod = "God Mode",
-    UI_Scoreboard_BanIp = "Ban Ip",
+    UI_Scoreboard_BanIp = "Ban IP,
```
Comment (Mishasama) [inline EN/UI_EN.txt:556]:
```suggestion
    UI_Scoreboard_BanIp = "Ban IP",
```

**Comment by Faalagorn** [comment]
> Please don't forget to check the code over and over again before committing the changes.

Thanks! I'm going to pull the changes you listed to my branch, but it seems they were present in the original string as well… Oddly enough just in English one, so thanks for noting them! https://github.com/TheIndieStone/ProjectZomboidTranslations/blob/1f9fdb19bbb9cdf40441308b902bd18b9185d133/EN/UI_EN.txt#L78

P.S. Did you look for them by hand, or did you use a parser for it? Also this PR will be cluttered with commits, partially by my inability to use git correctly especially on GitHub but hopefully no one will mind it. :grinning:

**Comment by Mishasama** [comment]
> > Please don't forget to check the code over and over again before committing the changes.
> 
> Thanks! I'm going to pull the changes you listed to my branch, but it seems they were present in the original string as well… Oddly enough just in English one, so thanks for noting them!
> 
> https://github.com/TheIndieStone/ProjectZomboidTranslations/blob/1f9fdb19bbb9cdf40441308b902bd18b9185d133/EN/UI_EN.txt#L78
> 
> P.S. Did you look for them by hand, or did you use a parser for it? Also this PR will be cluttered with commits, partially by my inability to use git correctly especially on GitHub but hopefully no one will mind it. 😀

Sorry, all by hand... So I said need to check it over and over again...😓

**Pull request merged**